docs/content/concepts/dagster-pipes/aws-emr.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: "Integrating AWS EMR with Dagster Pipes | Dagster Docs"
+description: "Learn to integrate Dagster Pipes with AWS EMR to launch external code from Dagster assets."
+---
+
+# AWS EMR & Dagster Pipes
+
+This tutorial gives a short overview on how to use [Dagster Pipes](/concepts/dagster-pipes) with [AWS EMR](https://aws.amazon.com/emr/).
+
+The [dagster-aws](/\_apidocs/libraries/dagster-aws) integration library provides the <PyObject object="PipesEMRClient" module="dagster_aws.pipes" /> resource, which can be used to launch AWS EMR jobs from Dagster assets and ops. Dagster can receive regular events such as logs, asset checks, or asset materializations from jobs launched with this client. Using it requires minimal code changes to your EMR jobs.
+
+---
+
+## Prerequisites
+
+- **In the orchestration environment**, you'll need to:
```

Comment:
Referring this to the "the orchestration environment" had cognitive overhead, where I had to think what you meant by this, when I think we could just say "In your Dagster environment install `dagster-aws`.

Reply:
ok made this change in all Pipes tutorials